docs/source/index.rst

```diff
@@ -6,36 +6,58 @@
 Substra documentation
 =====================
 
-Substra is a framework offering distributed orchestration of machine learning tasks among partners while guaranteeing secure and trustless traceability of all operations.
+Substra is an open source federated learning framework. It provides a flexible python interface and a web app to run federated learning training at scale.
 
-- :doc:`Getting started <get_started/overview>`
+Substra main usage is a production usage. It has already been deployed and used by hospitals and biotech companies. Yet Substra can also be used on a single machine on a virtually splitted dataset for two use cases:
 
-  An overview and guide to install Connnect python library.
+* to debug code before launching experiments on a real network
+* to perform FL simulations
```

Comment:
```suggestion
* debug code before launching experiments on a real network
* perform FL simulations
```

Reply:
This is more what SubstraFL does. More globally, the page seems to describe SubstraFL more than Substra.

Reply:
It depends what you mean by Substra, the product or the substra library.
Here it means the product, but I agree it adds a lot of confusion.